quodlibet/quodlibet/ext/events/equalizer.py

```diff
@@ -86,10 +87,25 @@ def interp_bands(src_band, target_band, src_gain):
 
 def get_config():
     try:
-        eq_levels_str = config.get('plugins', 'equalizer_levels')
-        return [float(s) for s in eq_levels_str.split(',')]
-    except (config.Error, ValueError):
-        return []
+        config_str = config.get('plugins', 'equalizer_levels')
```

Comment:
You can add a default value as the third argument, to prevent the `except` below from printing an error message the first time the plugin is used.

Reply:
Thanks, fixed. I added the disable buttons UI tweaks also.